svc/PrefService.ts

```diff
@@ -139,23 +132,25 @@ export class PrefService extends HoistService {
 
         if (isEmpty(updates)) return;
 
-        // clear obj state immediately to allow picking up next batch during async operation
         this._updates = {};
 
-        if (!isEmpty(updates)) {
-            await XH.fetchJson({
-                url: 'xh/setPrefs',
-                params: {
-                    updates: JSON.stringify(updates),
-                    clientUsername: XH.getUsername()
-                }
-            });
-        }
+        await XH.fetchJson({
+            url: 'xh/setPrefs',
+            params: {
+                updates: JSON.stringify(updates),
```

Comment:
Should we take this opportunity to move the updates to the body, as with the new bulk track call?